### PR TITLE
RID-4054 Upgrade Angular version

### PR DESCRIPTION
**Pull request by bartoszszafran**
This pull request updates the project's Angular version to 16.

**Changes:**
- Incrementally migrated the Angular versions from 9 to 16.
- Replaced TSLint with ESLint for linting.

**Important Note:**
When merging these changes, please do not squash the commits. Adi recommended adding tags for versions 9, 13, and 16, so we need these commits in master branch.

### CODE REVIEW

File: projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.spec.ts [hunk 1]
```diff
@@ -3,27 +3,27 @@ import { NgxTsRiskidService, RiskidSdkConfig, RISKID_SDK_CONFIG } from './ngx-ts
...
-          } as RiskidSdkConfig,
-        }
-      ]
-    });
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                SdkLoader,
+                NgxTsRiskidService,
+                {
```
Comment (adidim-transmit) [inline projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.spec.ts:13]: why those tabs were needed here generally on this PR?

File: projects/ngx-ts-riskid/package.json [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
   "repository": {
     "type": "git",
     "url": "git+https://github.com/TransmitSecurity/ngx-ts-riskid.git"
   },
   "homepage": "https://github.com/TransmitSecurity/ngx-ts-riskid/blob/main/projects/ngx-ts-riskid/README.md",
   "dependencies": {},
   "peerDependencies": {
-    "@angular/common": ">=9.x",
-    "@angular/core": ">=9.x"
+    "@angular/common": ">=10.x",
```
Comment (adidim-transmit) [inline projects/ngx-ts-riskid/package.json:14]: check that please

Comment (bartoszszafran) [reply projects/ngx-ts-riskid/package.json]: Nice catch, thanks :) It doesn't really seem to be used, but I upgraded the versions in https://github.com/TransmitSecurity/ngx-ts-riskid/pull/8/commits/f72052d1baff880af0a7d60e4b1c1aeb181db8ce :)

File: projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.spec.ts (hunk 1 above)
Comment (bartoszszafran) [reply projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.spec.ts:13]: These changes were made by ESLint after I migrated from TSLint to ESLint. TSLint was used in Angular<12, but it's no longer supported. There are two options:
1. The files were not correctly formatted before the migration.
2. The ESLint configuration generated by the migration tool might have some slight differences compared to the TSLint configuration.

I'm not sure it's worth the effort to align the new configs with the old ones, so I would just suggest to keep it as is :)

Comment (adidim-transmit) [reply projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.spec.ts]: cool!

**Review by adidim-transmit — APPROVED** [review]

**Pull request merged**